teemaderegister-be/src/controllers/users.js

```diff
@@ -174,8 +171,19 @@ module.exports.resetPicture = async (req, res) => {
     { new: true }
   ).select('profile.image.full updatedAt')
 
-  await fs.unlink(`${process.env.UPLOAD_DIR + original}`, err =>
-    log.debug(`${err && err.code} - Probably ${original} already deleted`))
+  await fs.unlink(`${process.env.UPLOAD_DIR + original}`, (err) =>
+    log.debug(`${err && err.code} - Probably ${original} already deleted`)
+  )
 
   return res.json({ user, message: 'Picture updated successfully' })
 }
+
+module.exports.getAllUsers = async (req, res) => {
+  const users = await User.find({})
```

Comment:
Here you can select or deselect the fields you do not need. I would exclude password. Others may be valid for admin to see.